framework/src/Volo.Abp.Quartz/Volo/Abp/Quartz/AbpQuartzModule.cs

```diff
@@ -20,9 +21,15 @@ public override void ConfigureServices(ServiceConfigurationContext context)
 
         public override void OnApplicationInitialization(ApplicationInitializationContext context)
         {
+            var options = context.ServiceProvider.GetRequiredService<IOptions<AbpQuartzPreOptions>>().Value;
+            
             _scheduler = context.ServiceProvider.GetService<IScheduler>();
             _scheduler.JobFactory = context.ServiceProvider.GetService<IJobFactory>();
-            _scheduler.Start();
+
+            if (options.StartDelay != null && options.StartDelay.Ticks > 0)
+                _scheduler.StartDelayed(options.StartDelay);
+            else
+                _scheduler.Start();
```

Comment:
I see `_scheduler.Start()` method return type is `Task`. This could be a problem.

Reply:
Yeah I noticed this too. @liangshiw any thoughts? I can only say that it hasn't caused any issues for me so far and that all Quartz and IJob exceptions have been caught and logged okay. The `Start()` method does have an exception that will be ignored if we discard the task.

This is the [QuartzScheduler.Start()](https://github.com/quartznet/quartznet/blob/master/src/Quartz/Core/QuartzScheduler.cs#L360) method. There is also a [RemoteScheduler.Start()](https://github.com/quartznet/quartznet/blob/master/src/Quartz/Impl/RemoteScheduler.cs#L209) method too.

Reply:
Resolved: https://github.com/abpframework/abp/pull/3470